src/plugins/recurring-issues/index.js

```diff
@@ -22,7 +22,9 @@ The next scheduled release will occur on ${releaseDate.format("dddd, MMMM Do, YY
 - [ ] Remove the 'tsc agenda' label on this issue
 - [ ] Review open pull requests and merge any that are [ready](https://eslint.org/docs/maintainer-guide/pullrequests#when-to-merge-a-pull-request)
 - [ ] Start the release on [Jenkins](https://jenkins.eslint.org)
-- [ ] Update the release blog post with a "Highlights" section
+- [ ] Update the release blog post:
+    - [ ] Add a "Highlights" section for any noteworthy changes.
+    - [ ] Remove the \`draft: true\` line in frontmatter.
```

Comment:
If we are adding fine-grained steps, perhaps we should also add “Replace authors: eslint with your GitHub username”?

Reply:
Good call! On an infinite time scale, I'd definitely forget to do that otherwise. 2cc715f